util/test/convert_start_test_log_to_junit_xml.py

```diff
@@ -108,6 +139,17 @@ def _parse_start_test_log(start_test_log):
         sub_test_lines = start_test_lines[subtest_start:subtest_end+1:1]
         del start_test_lines[:subtest_end+1]
 
+        subtest_error = None
+        subtest_file = None
+        if sub_test_lines[-1].startswith('[Error running sub_test'):
+            subtest_re = re.compile(r'\[Error running sub_test \(code [\d]+\) (?:in|for) (.*)\]')
+            match = subtest_re.match(sub_test_lines[-1])
```

Comment:
unless you hoist the compilation to a higher scope, there is no advantage to doing `re.compile` before calling `re.match`

Reply:
Do you happen to know the appropriate one-shot function?

Reply:
`re.match(r'\[Error running sub_test \(code [\d]+\) (?:in|for) (.*)\]', sub_test_lines[-1])`